Plugins/BackbonePluginTemplatePlugin/BackbonePluginTemplatePluginGenerator.cs

```diff
@@ -15,6 +16,7 @@ public class BackbonePluginTemplatePluginGenerator : IMetaGenerator
         private const string TestDirectorySuffix = "Test";
         private const string ProjectFileExtension = "csproj";
         private const string SolutionFileExtension = "sln";
+        private const string CentralPackageVersions = "Microsoft.Build.CentralPackageVersions";
```

Comment:
Rename this to `CentralPackageVersionsSdk`

Reply:
Addressed in: [Rename CentralPackageVersions constant](https://github.com/messerli-informatik-ag/meta-generator/pull/27/commits/cc4a78d7bb19f2fa0e086612734977e5dda5f59c)